src/screens/Dashboard/screens/Login/styles.module.scss

```
@@ -0,0 +1,90 @@
+@import 'scss/variables/_colors';
+@import 'scss/variables/_fonts';
+
+.app {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+}
+
+.head-container {
+  background-color: $cerulean;
+  height: 8px;
+  width: full;
+}
+.container {
+    background-color: $wild-sand;
+    height: 634px;
+    width: 300px;
+}
+
+.content{
+  margin: 10px 25px;
+}
+
+.input {
+  background-color: $white;
+  border-radius: 5px;
+  height: 32px;
+  width: 252px;
+}
+
+.text-label {
+  color: $black;
+  font-family: $Roboto;
+  font-size: 16px;
+  height: 22px;
+  margin: 5px 5px;
+  line-height: 24px;
+  text-align: left;
+  width: 100%;
+}
+
+.btn-primary{
+  background-color: $earls-green;
+  border-radius: 5px;
+  height: 32px;
+  margin-top: 15px;
+  width: 252px;
+}
+
+.btn-primary p {
```

avoid using html tags directly for class application
```suggestion
.btn-primary {
```